include/aws/common/exports.h

```diff
@@ -14,8 +14,8 @@
  * express or implied. See the License for the specific language governing
  * permissions and limitations under the License.
  */
-#if defined(USE_WINDOWS_DLL_SEMANTICS) || defined(WIN32)
-#    ifdef USE_IMPORT_EXPORT
+#if defined(AWS_COMMON_USE_WINDOWS_DLL_SEMANTICS) || defined(WIN32)
```

Comment:
These should probably be generic for all of the c libs or it's going to get unmanageable,

something like AWS_C_RT_USE_WINDOWS_DLL_SEMANTICS